org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/launch/ClasspathSeparationTest.groovy

```diff
@@ -0,0 +1,206 @@
+package org.eclipse.buildship.ui.launch
+
+import spock.lang.IgnoreIf
+import spock.util.environment.OperatingSystem
+
+import com.gradleware.tooling.toolingclient.GradleDistribution
+
+import org.eclipse.core.runtime.NullProgressMonitor
+import org.eclipse.debug.core.DebugPlugin
+import org.eclipse.debug.core.ILaunch
+import org.eclipse.debug.core.ILaunchConfiguration
+import org.eclipse.debug.core.ILaunchConfigurationType
+import org.eclipse.debug.core.ILaunchConfigurationWorkingCopy
+import org.eclipse.debug.core.ILaunchManager
+
+import org.eclipse.buildship.ui.test.fixtures.SwtBotSpecification
+
+// TODO (donat) enable the test when there's a Gradle snapshot release that includes the classpath separation changes
+// TODO (donat) adjust the Gradle distributions used by the test methods
+@IgnoreIf({ !OperatingSystem.current.isMacOs() })
+class ClasspathSeparationTest extends SwtBotSpecification {
+
+    def cleanup() {
+        DebugPlugin.default.launchManager.launchConfigurations.each { it.delete() }
+    }
+
+    def "Gradle doesn't supply scope information"() {
```

Comment:
❌ Please give tests a name that reflects what they are testing. "I test that Gradle doesn't supply scope information" is probably not what this test should be saying.

Reply:
Fixed